README.rst

```diff
@@ -29,9 +29,74 @@ Don't Panic
 Installation
 ^^^^^^^^^^^^
 
-TBD.
+To enable this product in a buildout-based installation:
```

Comment:
before I install the product, I want to know what problems are solved by it and why should I use it; how this is going to make my life easier?

it should include a paragraph describing why we're using webpack.